src/source/Ice/TurnConnection.c

```diff
@@ -1230,14 +1235,14 @@ STATUS turnConnectionTimerCallback(UINT32 timerId, UINT64 currentTime, UINT64 cu
 
     switch(pTurnConnection->state) {
         case TURN_STATE_GET_CREDENTIALS:
-            CHK_STATUS(iceUtilsSendStunPacket(pTurnConnection->pTurnPacket, NULL, 0, &pTurnConnection->turnServer.ipAddress,
-                                              pTurnConnection->pControlChannel, NULL, FALSE));
+            iceUtilsSendStunPacket(pTurnConnection->pTurnPacket, NULL, 0, &pTurnConnection->turnServer.ipAddress,
```

Comment:
Can we at least DLOGD or DLOGW on a failure to understand why this might fail? We need to have these points so later we can add metrics

Reply:
iceUtilsSendStunPacket already does CHK_LOG_ERR.

Reply:
That's true but it is hard to differentiate from the log which higher-level API the low-level API failure belongs to without reconstructing the log lines. These might be useful for connection failure deep dives and might give better clues when log diving. Think about adding it, also, this will be the anchor point for metrics later on